src/Release/GetReleaseToggles/ResponseToggle.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace Clearbooks\Labs\Release\GetReleaseToggles;
+
+class ResponseToggle
```

Comment:
I only want Use Cases to be dependant on the Toggle entity.  Exposing them to the consumers of the GetReleaseToggle use case would be violating Clean Architecture.

https://blog.8thlight.com/uncle-bob/2011/11/22/Clean-Architecture.html